ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -290,7 +290,7 @@ def _router_bgp_vlans_vlan(self, vlan, tenant) -> dict | None:
             "eos_cli": get(vlan, "bgp.raw_eos_cli"),
             "struct_cfg": get(vlan, "bgp.structured_config"),
         }
-        if self._evpn_gateway_vxlan_l2:
+        if self._evpn_gateway_vxlan_l2 and default(vlan.get("evpn_multi_domain"), tenant.get("evpn_multi_domain"), vrf.get("evpn_multi_domain"), True) is True:
```

Comment:
I would change the order to `vlan -> vrf -> tenant` rather than `vlan -> tenant -> vrf` or

Reply:
+1